pkgs/tools/misc/ppfetch/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ppfetch";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "yael333";
+    repo = "ppfetch";
+    rev = "acf00175113db7e4bcd04f40596494360d232fdf";
+    sha256 = "sha256-v1l+0MNY8aruoljf9jAzgrpgFiKgqdWTdLv4o78a8kU=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -Dm755 -t $out/bin ppfetch
+  '';
+
+  meta = with lib; {
+    description = " A colorful and prideful system info tool written in POSIX sh ";
```

According to section ["Meta attributes" in pkgs/README](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
`meta.description` should not start with a definite or indefinite article.
The implementation language seem less relevant to me.
If you believe there is more information about the utility that should be available in  Nix please consider adding `meta.longDescription`
```suggestion
    description = "Colorful and prideful system info tool";
```